torchvision/models/video/s3d.py

```diff
@@ -0,0 +1,214 @@
+from functools import partial
+from typing import Any, Callable, List, Optional
+
+import torch
+from torch import nn
+from torchvision.ops.misc import Conv3dNormActivation
+
+from ...transforms._presets import VideoClassification
+from .._api import register_model, Weights, WeightsEnum
+from .._meta import _KINETICS400_CATEGORIES
+from .._utils import _ovewrite_named_param
+
+
+__all__ = [
+    "S3D",
+    "S3D_Weights",
+    "s3d",
+]
+
+
+class TemporalSeparableConv(nn.Sequential):
+    def __init__(self, in_planes: int, out_planes: int, kernel_size: int, stride: int, padding: int = 0):
```

Comment:
It's a bit weird that we don't use the same `norm_layer` here as the rest of the network. I understand you do this because of different eps/momentum values? Worth clarifying that the values are indeed different by looking at the original TF implementation and on the other implementations. 

If it turns out this is needed, it might be worth we retrain the network to use the default eps/momentum and have a simpler/better API. Quite often we can simplify hyperparams without losing accuracy or compensate for the tiny drop by doing a newer optimization (augmentation, loss etc).

Reply:
it's actually the same `norm_layer` as the rest of the network, I just missed this module when i was making `norm_layer` configurable :)